docs/install-service/setup-openshift-390.md

```diff
@@ -0,0 +1,396 @@
+
+  $ oc create -f jenkins-pv.yml
+  ```
+- Bring up a Jenkins server using the jenkins persistent template. This has to
+  be done as normal user and not as admin user. ​ Make sure that the
+  Jenkins PV you created in above step is formated/emptied. ​ I spent about
+  2 days trying to debug issues arising out of stale data on that NFS storage.
+  Also add an extra environment variable to make sure that parallel builds work
+  as expected:
+
+  ```bash
+  $ oc login -u cccp
+  $ oc process -p MEMORY_LIMIT=1Gi openshift//jenkins-persistent | oc create -f -
+
+  # add env var for parallel builds
+  $ oc set env dc/jenkins JENKINS_JAVA_OVERRIDES="-Dhudson.slaves.NodeProvisioner.initialDelay=0,-Dhudson.slaves.NodeProvisioner.MARGIN=50,-Dhudson.slaves.NodeProvisioner.MARGIN0=0.85"
+
+  # add required permissions
+  $ oc login -u system:admin
+  $ oc adm policy add-scc-to-user privileged system:serviceaccount:cccp:jenkins
+  $ oc adm policy add-role-to-user system:image-builder system:serviceaccount:cccp:jenkins
+
+  ```
+
+Now go to the OpenShift web console and check if Jenkins deployment has come up
+and if the PVC of that deployment is bound to the PV you created earlier.
+
+To access Jenkins web console, you’ll need to check the OpenShift web console for the
+route that was created with nip.io subdomain. This should be visible on the
+project/namespace’s homepage on OpenShift console.
+
+Configuring DaemonSet: Scanning is one of the build pipeline phase the service
+offers. In scanning, we introspect the image built. In order to make scanning
+module available on all the possible builder nodes, we configure and deploy
+DaemonSet. The DeamonSet spins up a pod per builder node, which avails a docker
+volume for all the containers on the node. The scan stage in pipeline uses the
+volume for performing scan phase. DaemonSet needs to be deployed using cluster
+admin. Configure it with cluster admin user:
+
+```bash
+$ git clone https://github.com/dharmit/ccp-openshift
+$ cd ccp-openshift
+$ oc login -u system:admin
+$ oc create -f daemon-set/scan_data.yaml
+```
+
+Note: The labels defined for DaemonSet are used in pipeline seed-job/template.yaml
+to identify the container created using DaemonSet. Please keep the labels
+intact in DaemonSet template.
+
+Finally create the Jenkins Pipeline build to parse the container-index. This will create a
+seed-job which will parse the container index and create more Jenkins Pipeline builds
+that will create the actual container images for projects covered in the index. Replace
+172.29.33.8 with the IP address of your remote registry (you need to configure external
+registry by yourself, you can use the NFS VM created for same). Replace  master with the branch you’d like to deploy.
+
+```bash
+$ oc process -p PIPELINE_REPO=${PIPELINE_REPO} -p PIPELINE_BRANCH=${PIPELINE_BRANCH} -p REGISTRY_URL=${REGISTRY_URL} -p NAMESPACE=`oc project -q` -f seed-job/buildtemplate.yaml | oc create -f -
```

Comment:
Can you replace this with below for readability? 

```bash
oc process -p PIPELINE_REPO=${PIPELINE_REPO} \
  -p PIPELINE_BRANCH=${PIPELINE_BRANCH} \
  -p REGISTRY_URL=${REGISTRY_URL} \ 
  -p NAMESPACE=`oc project -q` \ 
  -f seed-job/buildtemplate.yaml | oc create -f -
```

More importantly, `seed-job/buildtemplate.yaml` has many more parameters now which we must account for. This command is not enough as a part of docs even if it's enough to bring up the service with our chosen defaults.